content/docs/command-reference/metrics/index.md

```diff
@@ -10,24 +10,37 @@ A set of commands to add, manage, collect, and display project metrics:
 ## Synopsis
 
 ```usage
-usage: dvc metrics [-h] [-q | -v] {show,add,modify,remove} ...
+usage: dvc metrics [-h] [-q | -v] {show,add,modify,remove,diff} ...
 
 positional arguments:
-  COMMAND
-    show                Output metric values.
-    add                 Tag file as a metric file.
-    modify              Modify metric file values.
-    remove              Remove files's metric tag.
+  {show,add,modify,remove,diff}
+                        Use `dvc metrics CMD --help` to display command-
+                        specific help.
+    show                Print metrics, with optional formatting.
+    add                 Mark a DVC-tracked file as a metric.
+    modify              Modify metric default formatting.
+    remove              Remove metric mark on a DVC-tracked file.
+    diff                Show changes in metrics between commits
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -q, --quiet           Be quiet.
+  -v, --verbose         Be verbose
 ```
 
 ## Description
 
-DVC has the ability to mark a certain stage <abbr>outputs</abbr> as files
-containing metrics to track. (See the `--metrics` option of `dvc run`.) Metrics
-are project-specific numeric values e.g. `AUC`, `ROC`, etc. DVC itself does not
-ascribe any specific meaning for these numbers. Usually these numbers are
-produced by the model evaluation script and serve as a way to compare and pick
-the best performing experiment.
+In order to track metrics associated to machie learning experiments, DVC has the
+ability to mark a certain stage <abbr>outputs</abbr> as files containing metrics
+to track. (See the `--metrics` option of `dvc run`.) Metrics are
+project-specific floating-point values e.g. `AUC`, `ROC`, etc.
+
+Supported file formats: JSON. Metrics can be organized in a tree hierarchy in a
+JSON file. DVC addresses the metrics by the tree path.
```

Comment:
List of a single item seems strange.

Reply:
"tree hierarchy"/"tree path" concepts need more explaining, maybe a quick example, or simplifying the text. Or even removing that part, since it's better explained in the metrics add/show cmd refs.

Reply:
> List of a single item seems strange

p.s. I know YAML will be added but will it be soon? Prob better to have this as a sentence such as "Currently, DVC supports metric files in JSON format."

Reply:
Yes, it should be added soon. Let's keep it here for a week or two :)